content/blog/cdf-intro-uber-crane/index.md

```diff
@@ -5,9 +5,9 @@ author: "Chen Wang"
 description: "Real World FABR: w.r.t. Crane, Uber's Next-Gen infrastructure Stack"
 ---
 
-*This is part of the series introducing FABR, including OpenFABR CDF and FABR infrastructure, in real-world scenarios where actionable recommendations can be applied to solve infrastructure challenges.*
+*This is part of the series introducing FABR, including [OpenFABR CDF](https://github.com/openfabr/cdf) and [FABR Infra](https://fabrhq.com), in real-world scenarios where actionable recommendations can be applied to solve infrastructure challenges.*
 
-TODO: add a title image with CDF + Uber Crane
+TODO: add a title image with FABR + Uber Crane
```

Comment:
Would appreciate the help of a hand-drawn style title image here

Reply:
I'll have a quick try, see what comes out.

Reply:
I've added an image (DELL-E)